support-frontend/assets/pages/digital-subscription-landing/digitalSubscriptionLanding.jsx

```diff
@@ -93,9 +93,9 @@ type State = {
 // ----- Render ----- //
 class LandingPage extends Component<Props, State> {
   state = {
-    showPage: this.props.optimizeHasLoaded,
+    showPage: true, // this.props.optimizeHasLoaded,
```

Comment:
This prevents the page waiting for Optimize tests as there are no longer any on this page

Reply:
Yes, I started removing it all and then thought it was probably a waste of time to do that.